label_studio/ml_model_providers/models.py

```diff
@@ -59,29 +61,3 @@ def has_permission(self, user):
         return (
             user.is_administrator or user.is_owner or user.is_manager
         ) and user.active_organization_id == self.organization_id
-
-    def validate_api_key(self):
```

Comment:
Why was this funtionality moved to lse_ml_models/serializers.py in separate functions? e.g. `def _validate_openai_connection`